README.md

```diff
@@ -86,6 +86,49 @@ Found 1 pipelines: customers_orders (python)
 
 ✅ Unit test for pipeline: pipelines/customers_orders succeeded.
 ```
+To quickly validate if all your pipelines are not broken 
```

Comment:
Add doc to prophecy-docs as well

Reply:
https://github.com/SimpleDataLabsInc/prophecy-docs/pull/102